ps/jenkins/test-ps-innodb-cluster.yml

```diff
@@ -14,9 +14,9 @@
     pipeline-scm:
         scm:
             - git:
-                url: https://github.com/Percona-Lab/jenkins-pipelines.git
+                url: https://github.com/kaushikpuneet07/jenkins-pipelines.git
```

Comment:
before merge please set correct repos and branches

Reply:
fixed